randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -2566,7 +2567,9 @@ Extra - asset_id: collision_camera_031
   > Door to EMMI Zone Exit East Access
       Can Slide
   > Tunnel to Purple EMMI Arena
-      Simple IBJ or Use Spin Boost
+      Any of the following:
+          Flash Shift or Walljump (Beginner) or Simple IBJ or Use Spin Boost
+          Slide and Slide Jump (Beginner)
```

Comment:
Is the walljump from the ledge grab? do you think it's hard enough to be in a trick?

Reply:
It's not hard..But I think that most of the people won't think of a ledge grab to get up there. I would leave it with the trick just to set a difference between first time players and people who already played more seeds.